test/models/network-v2.spec.ts

```diff
@@ -65,45 +65,141 @@ describe(`NetworkV2`, () => {
 
     describe(`Owner`, () => {
       it(`changeCouncilAmount()`, async () => {
-        await hasTxBlockNumber(network.changeCouncilAmount(newCouncilAmount));
-        expect(await network.councilAmount()).to.eq(newCouncilAmount.toString());
+        await hasTxBlockNumber(network.changeCouncilAmount(103000));
+        expect(await network.councilAmount()).to.eq((103000).toString());
       });
 
+      it(`changeCouncilAmount emits event`, async () => {
```

Comment:
I always try to have one exception in one it statement. 
Therefore I created the 'event emitted' it clauses. (It's also clearer what value it changed from and to)

Reply:
cool! :) you can, however, use `expectEvent` (outputed from `test/utils/index.ts`)